### ISSUE TITLE
workspace/configuration and GNOME Builder

### ISSUE THREAD

**Issue by Prince781** [benwaffle/vala-language-server#101]
Support `workspace/configuration` request from GNOME Builder.

**Comment by JCWasmx86** [comment]
This would basically be the matter of connecting to the `load-configuration`-signal. I can write a patch, if you can give me some more details, e.g. where should the configuration be? E.g. a configuration file (.json) or something else. Furthermore I have seen that VLS does not even use `workspace/configuration`

**Comment by Prince781** [comment]
Yes. It's possible for VLS to get project information from Builder instead of introspecting the build files. Then, when evaluating dependencies, Builder can point us to the SDK it downloaded rather than us having to search the system.
